contractcourt/briefcase.go

```diff
@@ -1506,3 +1527,106 @@ func decodeCommitSet(r io.Reader) (*CommitSet, error) {
 
 	return c, nil
 }
+
+func encodeTaprootAuxData(w io.Writer, c *ContractResolutions) error {
+	tapCase := newTaprootBriefcase()
+
+	if c.CommitResolution != nil {
+		commitResolution := c.CommitResolution
+		commitSignDesc := commitResolution.SelfOutputSignDesc
+		tapCase.CtrlBlocks.CommitSweepCtrlBlock = commitSignDesc.ControlBlock
```

Comment:
This is a nil deference?

Reply:
don't think so since `CtrlBlocks` is init'd earlier in `newTaprootBriefcase`, unless you meant `commitSignDesc.ControlBlock`?